docs/src/Checkbox.doc.js

```diff
@@ -327,16 +327,28 @@ card(
     disabled={[false, true]}
     indeterminate={[false, true]}
     size={['sm', 'md']}
+    hasCheckerboard={false}
   >
     {(props, i) => (
-      <Checkbox id={`example-${i}`} onChange={() => {}} {...props} />
+      <Checkbox
+        id={`example-${i}`}
+        accessibilityLabelledBy={`combinations-label-${i}`}
```

Comment:
@ayeshakmaz we've been trying to avoid using `accessibilityLabelledBy` and use `id` + `<Label htmlFor />` instead. 

Looks like we only use `accessibilityLabelledBy` in the `<Combination />`.